nodes/extensions/admin/pulp_node/extensions/admin/rendering.py

```diff
@@ -36,7 +25,7 @@
 
 ACTIONS = {
     RepositoryReport.PENDING: _('Pending'),
-    RepositoryReport.CANCELLED: _('Cancelled'),
+    RepositoryReport.CANCELLED: _('Canceled'),
```

Comment:
Note that the variable is still spelled "CANCELLED" - it might be best to keep that consistent.